src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1310,15 +1314,17 @@ namespace ReportSizingManager {
 						} else {
 							if ( CurDuctType == Main ) {
 								if ( SameString ( CompType, "COIL:HEATING:WATER" ) ) {
-									if (FinalSysSizing( CurSysNum ).SysAirMinFlowRat > 0.0) {
+									if ( FinalSysSizing( CurSysNum ).SysAirMinFlowRat > 0.0 && !DataDesicRegCoil ) {
 										AutosizeDes = FinalSysSizing( CurSysNum ).SysAirMinFlowRat * FinalSysSizing( CurSysNum ).DesMainVolFlow;
+									} else if ( DataDesicRegCoil ) {
+										AutosizeDes = FinalSysSizing( CurSysNum ).DesMainVolFlow;
```

Comment:
redundant else if since the first if tests for DataDesRegCoil and the else sets the same value. I guess it can't hurt to be specific. Now in this section, using DesMainVolFlow is correct since this is a heating coil that is not in the OA system.